pkgs/applications/editors/jupyter-kernels/coq/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, callPackage
+, runCommand
+, makeWrapper
+, coq
+, imagemagick
+, python3
+}:
+
+# To test (in root nixpkgs dir):
+# $(nix-build -E 'with import ./. {}; jupyter.override { definitions = { coq = coq-kernel.definition; }; }')/bin/jupyter-notebook
+
+# To test with packages:
+# $(nix-build -E 'with import ./. {}; jupyter.override { definitions = { coq = coq-kernel.definitionWithPackages [coqPackages.ceres]; }; }')/bin/jupyter-notebook
+
+let
+  kernel = callPackage ./kernel.nix {};
+
+in
+
+rec {
+  launcher = runCommand "coq-kernel-launcher" {
+    inherit coq;
+    python = python3.withPackages (ps: [ ps.traitlets ps.jupyter_core ps.ipykernel kernel ]);
+    buildInputs = [ makeWrapper ];
+  } ''
+    mkdir -p $out/bin
+
+    makeWrapper $python/bin/python $out/bin/coq-kernel \
+      --add-flags "-m coq_jupyter" \
+      --suffix PATH : $coq/bin
+  '';
+
+  sizedLogo = size: stdenv.mkDerivation {
+    pname = "coq-logo-${size}x${size}.png";
+    inherit (coq) version;
+
+    src = coq.src;
+
+    buildInputs = [ imagemagick ];
+
+    dontConfigure = true;
+    dontInstall = true;
+
+    buildPhase = ''
+      convert ./ide/coqide/coq.png -resize ${size}x${size} $out
+    '';
```

Comment:
Please generate all sizes in one derivation. Also this should use runCommandNoCC

Reply:
Huh? `runCommandNoCC` isn't used all in the current master branch of Nixpkgs. It's aliased to `runCommand`. I did it but I think I'd better change it back if you don't mind.

All sizes are in one derivation now.

Reply:
I've changed it back in my latest commit, lmk if you want to discuss further.